documenters_aggregator/spiders/cpsboe.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import scrapy
+
+from datetime import datetime
+from pytz import timezone
+
+class CpsboeSpider(scrapy.Spider):
+    name = 'cpsboe'
+    allowed_domains = ['www.cpsboe.org']
+    start_urls = ['http://www.cpsboe.org/meetings/planning-calendar']
+    domain_url = 'http://www.cpsboe.org'
+
+    def parse(self, response):
+        for item in response.css('#content-primary tr')[1:]:
+            yield {
+                '_type': 'event',
+                'id': self._parse_id(item),
+                'name': 'Chicago Board of Education Monthly Meeting',
+                'description': self._parse_description(item),
+                'classification': self._parse_classification(item),
+                'start_time': self._parse_start_time(item),
+                'all_day': self._parse_all_day(item),
+                'status': self._parse_status(item),
+                'location': self._parse_location(item)
+
+            }
+
+    def _parse_id(self, item):
```

Comment:
There should only be one event in a day on the planning calendar for regular board meetings. If you get to actual/all board meetings, there can be multiple meetings on the same day ([committee meetings](http://www.cpsboe.org/meetings/details/238), [public](http://www.cpsboe.org/meetings/details/212) hearings, [budget](http://www.cpsboe.org/meetings/details/251) meetings, etc).

See past meetings for examples: http://www.cpsboe.org/meetings/past-meetings